ui/src/routes/guides/_layout.svelte

```diff
@@ -0,0 +1,198 @@
+<script>
+	import * as _ from 'lamb';
+	import {_screen}
+		from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
+
+	import ChevronLeft from '@svizzle/ui/src/icons/feather/ChevronLeft.svelte';
+	import ChevronRight from '@svizzle/ui/src/icons/feather/ChevronRight.svelte';
+	import Icon from '@svizzle/ui/src/icons/Icon.svelte';
+	import Link from '@svizzle/ui/src/Link.svelte';
+
+	import {isNotNil} from '@svizzle/utils';
+	import {toolName} from 'app/config';
+	import theme from 'app/theme';
+
+	const links = {
+		indicators: 'Indicators',
+		a11ymenu: 'A11y menu',
+		app:'App',
+	};
+	const segments = _.keys(links);
```

Comment:
(*) The order should be `['app', 'indicators', 'a11ymenu']` so here we might want to have something like:

```suggestion
	const links = {
		indicators: {label: 'Indicators', order: 1},
		a11ymenu: {label: 'A11y menu', order: 2},
		app: {label: 'App', order: 0},
	};
	const segments = _.sort(_.keys(links), [_.getKey('order')]);
```